.github/workflows/tact.yml

```diff
@@ -57,14 +57,14 @@ jobs:
       - name: Compare Tact version from CLI flag `--version` against package.json
         if: runner.os != 'Windows'
         run: |
-          if [ "$(./bin/tact --version)" != "$(jq -r '.version' < package.json)" ];
+          if [ "$(./bin/tact.mjs --version)" != "$(jq -r '.version' < package.json)" ];
```

Comment:
Would be nice to remove this check and reuse `getCompilerVersion` function in the CLI module